lib/openstax/aws/stack.rb

```diff
@@ -325,6 +325,11 @@ def self.format_hash_as_tag_parameters(tags)
       tags.map{|tag| {key: tag.key, value: tag.value}}
     end
 
+    def self.list_stacks(region: "us-east-1")
+      client = Aws::CloudFormation::Client.new(region: region)
+      client.list_stacks.map { |response| response.stack_summaries.map { |stack| stack.stack_name } }.flatten
```

Comment:
Hey @PiotrKozlowski - thanks for diving in to this!!

I think the `list_stacks` command needs a set of status filters, otherwise it can return stacks that failed to create or stacks that have been deleted.

I also think there's probably some benefit to this method handling multiple regions and handling the stack name filtering so that unified-deployment doesn't have to do that in multiple places.

Also, if we return instances of `OpenStax::Aws::Stack` instead of just the stack name, we'll have access to more metadata.

Take a look at the alternative implementation I made in https://github.com/openstax/aws-ruby/pull/56